config/default/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
this might pose problem for local set up where people load the image to the local kind cluster using 'kind load-image'.

see https://github.com/kubernetes-sigs/cluster-api/pull/4220/files#diff-7d6aeba78cbb198ada9c37bb5c9deaf40948598ff8df831955c6118093e0fa63R107

Reply:
Thx. That one was not intended :/ The conformance test changes this on every run so sometimes I accidentally check it in.

Reply:
Fixed